src/main/joyride/sci.cljs

```diff
@@ -20,7 +22,9 @@
            (sci/init {:classes {'js goog/global
                                 :allow :all}
                       :namespaces (assoc (:namespaces pconfig/config)
-                                         'joyride.core {'*file* sci/file})
+                                         'joyride.core {'*file* sci/file
+                                                        'get-extension-context (fn []
```

Comment:
PR looks good, but I'd prefer if get-extension context was a "true" var inside the SCI config. You can create it by defining `get-extension-context` with `defn` and then use `(sci/copy-var get-extension (sci/create-ns 'joyride.core))`

Reply:
This would also allow you to add a docstring to it.

Reply:
And get it using `clojure.repl/doc`.

Reply:
Thanks! Where would I do the copying?

Reply:
Look around in nbb.core for copy-var and you'll find an example. Sorry, tight schedule today!

Reply:
I had already started poking around there. I think I have fixed it in a proper way now. And doc lookup works as well, added it to the Joyride API example while at it.

Good to merge?